### DISCUSSION TITLE
텔레포트가 잘 안됩니다

### DISCUSSION THREAD

**Discussion by smilejsu82** [naverz/zepeto-studio-kor#556] [5. Q&A]
월드 1.1.5 사용중입니다 

플레이어가 죽었을때 죽는 애니메이션후 특정 위치로 텔레포트 하려고 합니다 
죽었을때 코드 입니다 
![image](https://user-images.githubusercontent.com/50188795/160800093-ff1a2987-d553-4aa5-abf6-088db83ee534.png)

이후 죽는 애니메이션 1.9초 이후 텔레 포트 합니다 
![image](https://user-images.githubusercontent.com/50188795/160800196-8b630cdb-f71b-4097-928d-cf72119a7fee.png)

간헐적으로 죽은 위치에서 바로 텔레포트 되는데 무엇을 잘못 하고 있는건가요?

현재 월듸 1.1.5 사용중입니다, 땅속에 박히는 문제는 이전글을 참고했는데 고쳐졌나요?

https://github.com/naverz/zepeto-studio-kor/discussions/371

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. 
텔레포트 사용시 땅속에 묻히는 부분은 resurrectionPoint 오브젝트의 좌표값을 살펴보면 좋을 것 같습니다.
this.zepetoCharacter.Teleport(new Vector3(0, 0, 0), Quaternion.identity);
이렇게 사용할 경우 정상적으로 착지가 되는 것을 확인할 수 있습니다.

**Reply by smilejsu82** [reply]
resurrectionPoint 오브젝트의 좌표값은 이상 없었습니다 
![image](https://user-images.githubusercontent.com/50188795/160971997-fe34258e-d291-457c-aa51-7b19581ba5bd.png)

이전 게시글을 참고로 Teleport State에 아무 Clip이나 넣었더니 해결 되었습니다 
![image](https://user-images.githubusercontent.com/50188795/160972080-ac64f66e-a3a5-44aa-b176-0cad0467d06e.png)
![image](https://user-images.githubusercontent.com/50188795/160971972-b7fa1bec-aa69-4eee-8eb6-f1aa325ac4d3.png)
![image](https://user-images.githubusercontent.com/50188795/160972052-b74cf39f-2502-4b5f-a041-95a83f439b8b.png)


하지만 가끔 제자리로 텔레포트 되는 현상이 있습니다 

https://user-images.githubusercontent.com/50188795/160972301-0a12ccf5-8faf-4470-922a-3675cb537727.mp4


죽었을때 코드 
![image](https://user-images.githubusercontent.com/50188795/160972348-8532599b-301f-4083-82a5-b78a1377adfc.png)

죽는 애니메이션 끝날때까지 대기후 텔레포트 
![image](https://user-images.githubusercontent.com/50188795/160972332-49c300bb-dffa-4432-88c6-b92be2b5438b.png)

**Reply by Zepeto-TechEvangelist** [reply]
땅속에 파묻히는 이슈는 해결이지만, 
간헐적으로 텔레포트가 동작하지 않는 문제로 정리해도 될까요?

**Reply by smilejsu82** [reply]
감사합니다 땅속에 파묻히는 이슈는 Teleport State에 임의 클립을 넣어서 해결 되었습니다
